core/src/main/resources/lib/form/entry.jelly

```diff
@@ -89,8 +89,9 @@ THE SOFTWARE.
         <div class="setting-main">
```

Comment:
The comment on the two lines above the one I'm commenting on should be deleted, with https://github.com/jenkinsci/jenkins/pull/6460/files#r867286362 it makes no sense anymore.